api/src/main/webapp/static/aou-file-tree-policy-extension.js

```diff
@@ -6,30 +6,62 @@ define([
 ], (Jupyter) => {
 
   const load_upload_extension = () => {
-  // This will open a dialog box when the user clicks Upload Button on the Jupyter File List page.
-  // File will be uploaded only if the user clicks OK else it will just close the dialog box and do nothing.
-
+    // This will open a dialog box when the user clicks Upload Button on the Jupyter File List page.
+    // File will be uploaded only if the user clicks OK else it will just close the dialog box and do nothing.
     $('#notebook_list_info input').click(() => confirm(
         'It is All of Us data use policy to not upload data or files containing ' +
         'personally identifiable information (PII). Any external data, files, or software that ' +
         'is uploaded into the Workspace should be exclusively for the research purpose that was ' +
         'provided for this Workspace.'));
 
-    // notebook-list.js in the Jupyter UI serves the download by using it's own
+    // notebook-list.js in the Jupyter UI serves the download and view button by using it's own
     // jQuery click handler. We have no way of ordering our click handler ahead
     // of it without digging into jQuery internals. Instead, we insert a span
     // on top of the button to gain click priority.
     $('.dynamic-buttons .download-button').empty().append(
         '<span id="aou-download-button-overlay">Download</span>');
+    $('.dynamic-buttons .view-button').empty().append(
+        '<span id="aou-view-button-overlay">View</span>');
+
+
+    function downloadPolicyPopUp() {
```

Comment:
i am not sure if this is the right spot for this function, open for suggestion

Reply:
LGTM